packages/metrics/src/features/metrics/metricsGenerator.ts

```diff
@@ -399,12 +421,18 @@ export async function fetchKeyFigures(
       FROM birth_registration
     WHERE time >= ${timeStart}
       AND time <= ${timeEnd}
-      AND ( locationLevel2 = '${queryLocationId}'
-          OR locationLevel3 = '${queryLocationId}'
-          OR locationLevel4 = '${queryLocationId}'
-          OR locationLevel5 = '${queryLocationId}' )
+      AND ( locationLevel2 = $queryLocationId
+          OR locationLevel3 = $queryLocationId
+          OR locationLevel4 = $queryLocationId
+          OR locationLevel5 = $queryLocationId )
       AND ageInDays <= ${EXPECTED_BIRTH_REGISTRATION_IN_DAYS}
-    GROUP BY gender`
+    GROUP BY gender`,
+    {
+      placeholders: {
+        queryLocationId,
+        EXPECTED_BIRTH_REGISTRATION_IN_DAYS
```

Comment:
I don't think we need `EXPECTED_BIRTH_REGISTRATION_IN_DAYS` inside the placeholders as it's still being directly interpolated in the string literal.